packages/bp/src/jest-before.ts

```diff
@@ -7,6 +7,7 @@ const { Debug: _Debug } = require('./debug.ts')
 
 global.DEBUG = _Debug
 
+require('dotenv').config()
```

Comment:
how is that necessary ? do you have a .env for your tests ?

Reply:
The postgres tests look into the environment for the DB url / username, password, port, etc. They weren't being loaded so I added this line in case you have a non default postgres setup.